packages/framework/src/Resources/views/Admin/Form/theme.html.twig

```diff
@@ -663,7 +663,7 @@
                 <ul class="in-message__list">
                     {% if errors|length > 0 %}
                         {% for error in errors %}
-                            <li class="js-validation-errors-message js-error-source-id-{{ getFormErrorSourceId(error) }} }}">
+                            <li class="in-message__list__item js-validation-errors-message js-error-source-id-{{ getFormErrorSourceId(error) }}">
```

Comment:
this added class is useless right? can you please remove it?

Reply:
yes after my last changes is useless. Removed